src/main/java/com/salesforce/storm/spout/sideline/handler/SidelineSpoutHandler.java

```diff
@@ -127,9 +127,6 @@ public void onSpoutOpen(DynamicSpout spout, Map topologyConfig, TopologyContext
             null
         );
 
-        // Our main firehose spout instance.
```

Comment:
Instead of adding the fire hose immediately, lets wait until we've applied any filters that should be active when it starts.